client/src/utils/constants.js

```diff
@@ -1 +1,5 @@
 // Store constants here for reusability across application
+import dotenv from "dotenv";
+dotenv.config();
```

Comment:
We should import and run this at the top level of the app, not in our constants file